packages/gatsby-plugin-mdx/src/gatsby-layout-loader.ts

```diff
@@ -127,8 +137,6 @@ const gatsbyLayoutLoader: LoaderDefinition = async function (
         })
       }
 
-      newBody.push(declaration)
```

Comment:
It didn't seem useful that we added this (also see snapshot changes to see the change) so I just removed it (?)

Reply:
Test failure probably due to this https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/85768/workflows/0b9e1580-4191-4534-bb2d-9af4ebbeb255/jobs/1029232